src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -216,97 +325,5459 @@ static ColliderJntSphInit D_80B7A888 = {
+
+    if (D_80B7FDA8 != 0) {
+        D_80B7FDA8--;
+
+        Math_ApproachF(&D_80B7A6C0, 35.0f, 1.0f, 100.0f);
+        Math_ApproachF(&D_80B7A6BC, -0.8f, 1.0f, 0.4f);
+        Math_ApproachS(&player->actor.shape.rot.x, -4000, 2, 15000);
+    } else {
+        s16 target = 0;
+
+        if ((D_80B7A694 == 4) && (D_80B7E124 != 0)) {
+            target = Math_SinS(D_80B7E0AE * 25600) * 1500.0f;
+        } else {
+            Math_ApproachZeroF(&D_80B7A6C0, 0.1f, 10.0f);
+            Math_ApproachZeroF(&D_80B7A6BC, 1.0f, 0.05f);
+        }
+
+        Math_ApproachS(&player->actor.shape.rot.x, target, 5, 1000);
+    }
+
+    if ((D_80B7A694 == 3) || (D_80B7A694 == 4)) {
+        if ((input->rel.stick_x == 0) && (D_80B7A6C4 != 0)) {
+            D_80B7A6B0 = 0.0f;
+        }
+        if ((input->rel.stick_y == 0) && (D_80B7A6C8 != 0)) {
+            D_80B7A6B4 = 0.0f;
+        }
+
+        spC8 = player->unk_85C;
+        Math_SmoothStepToF(&player->unk_85C, input->rel.stick_y * 0.02f, 0.3f, 5.0f, 0.0f);
+        spC8 = player->unk_85C - spC8;
+
+        spC4 = player->unk_858;
+        Math_SmoothStepToF(&player->unk_858, input->rel.stick_x * 0.02f, 0.3f, 5.0f, 0.0f);
+        spC4 = player->unk_858 - spC4;
+
+        if (player->unk_858 > 1.0f) {
+            player->unk_858 = 1.0f;
+        }
+        if (player->unk_85C > 1.0f) {
+            player->unk_85C = 1.0f;
+        }
+        if (player->unk_858 < -1.0f) {
+            player->unk_858 = -1.0f;
+        }
+        if (player->unk_85C < -1.0f) {
+            player->unk_85C = -1.0f;
+        }
+
+        Math_ApproachF(&D_80B7A6A8, spC4 * 70.0f * -0.01f, 1.0f, D_80B7A6B0);
+        Math_ApproachF(&D_80B7A6B0, 1.0f, 1.0f, 0.1f);
+        Math_ApproachF(&D_80B7A6AC, spC8 * 70.0f * 0.01f, 1.0f, D_80B7A6B4);
+        Math_ApproachF(&D_80B7A6B4, 1.0f, 1.0f, 0.1f);
+        Math_ApproachZeroF(&D_80B7A6B8, 1.0f, 0.05f);
+    } else {
+        Math_ApproachZeroF(&player->unk_85C, 1.0f, 0.1f);
+        Math_ApproachZeroF(&player->unk_858, 1.0f, 0.1f);
+        Math_ApproachF(&D_80B7A6AC, (Math_SinS(D_80B7E0AE * 3000) * 0.025f) + -0.03f, 1.0f, 0.05f);
+        Math_ApproachZeroF(&D_80B7A6A8, 1.0f, 0.05f);
+
+        if ((D_80B7E0B4 >= 19) && (D_80B7E0B4 <= 24)) {
+            Math_ApproachF(&D_80B7A6B8, 0.8f, 1.0f, 0.2f);
+        } else {
+            Math_ApproachF(&D_80B7A6B8, 0.0f, 1.0f, 0.4f);
+        }
+    }
+
+    func_80093D18(globalCtx->state.gfxCtx);
+
+    gSPDisplayList(POLY_OPA_DISP++, gFishingRodSetupDL);
+
+    gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 255, 155, 0, 255);
+
+    Matrix_Mult(&player->mf_9E0, MTXMODE_NEW);
+
+    if (sLinkAge != 1) {
+        Matrix_Translate(0.0f, 400.0f, 0.0f, MTXMODE_APPLY);
+    } else {
+        Matrix_Translate(0.0f, 230.0f, 0.0f, MTXMODE_APPLY);
+    }
+
+    if (D_80B7A694 == 5) {
+        Matrix_RotateY(1.7592919f, MTXMODE_APPLY);
+    } else {
+        Matrix_RotateY(1.288053f, MTXMODE_APPLY);
```

Comment:
```suggestion
        Matrix_RotateY(DEG_TO_RAD(100.8f), MTXMODE_APPLY);
    } else {
        Matrix_RotateY(DEG_TO_RAD(73.8f), MTXMODE_APPLY);
```
I guess it's slightly better than radians

Reply:
We don't write constants as degrees currently, so I think this should be discussed separately and we should apply it globally if we decide to do it.